docker/postgres/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+pg_restore -U ${POSTGRES_USER} -d ${POSTGRES_DB} --no-owner /var/lib/postgresql/data/baseregisters.tar
```

Comment:
This is not the final form of the entrypoint. Right now I am importing many different databases:
baseregisters_test = only empty tables
baseregisters_old = balazs database from the 3dbag-sample-data image
baseregisters_empty = just an empty one created here
and of course dagster = for dagster to save its metadata.

Reply:
NO reason actually, I just wanted to make it work like this just to test what happens if I delete the container etc. I can remove it later, I will just keep while I am still doing these tests.